src/api/configurator.ts

```diff
@@ -61,7 +61,12 @@ export async function makeConfig(): Promise<IConfigFunction>
 				name: "sports"
 			},
 			{
-				name: "random"
+				name: "random",
+				types: (() => { let arr = []; for (let i = 0; i < 10000; i++) { arr[i] = `${i}` } return arr })(),
```

Comment:
Hope, this is temporary solution :)

Reply:
Definitely. We're going to add parametrized data requests soon.